roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -167,6 +167,7 @@ istio_enabled: false
 registry_enabled: false
 enable_network_policy: false
 local_volume_provisioner_enabled: false
+cephfs_provisioner_enabled: false
 persistent_volumes_enabled: false
 cephfs_provisioner_enabled: false
```

Comment:
was added yesterday already. 
this PR is adding cephfs_provisioner_storage_class var only

Reply:
O_O|| sorry I miss this change... So now just keep the change for cephfs_provisioner_storage_class